packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -450,66 +450,65 @@
                         </oui-action-menu>
                     </oui-tile-definition>
                     <oui-tile-definition
-                        data-term="{{ ::'hosting_dashboard_service_privatesql' | translate }}"
+                        data-term="{{:: 'hosting_dashboard_service_privatesql' | translate }}"
                     >
                         <oui-tile-description>
-                            <div
-                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseActive > 0"
-                                data-wuc-progress-bar-element-counter
-                                data-wuc-pbec-nb-elements="hosting.sqlPriveInfo.nbDataBaseActive"
-                                data-wuc-pbec-max-elements="hosting.sqlPriveInfo.nbDataBaseInclude"
-                                data-wuc-pbec-upper-limit="cloudWebUnlimitedQuantity"
-                            ></div>
                             <p
-                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude === 0"
-                            >
-                                0
-                            </p>
+                                data-ng-if="!hosting.sqlPriveInfo.hasPrivateDatabaseToActivate"
+                                data-ng-bind="privateDatabases.length"
+                            ></p>
+
                             <p
-                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude > hosting.sqlPriveInfo.nbDataBaseActive"
+                                data-ng-if="hosting.sqlPriveInfo.hasPrivateDatabaseToActivate"
                                 data-translate="hosting_dashboard_service_privatesql_active_remaining"
-                                data-translate-values="{
-                                   count: hosting.sqlPriveInfo.nbDataBaseInclude - hosting.sqlPriveInfo.nbDataBaseActive,
-                               }"
+                                data-translate-values="{ count: 1 }"
```

Comment:
Are you sure it's always one? Then maybe the sentence should be rewritten to something like : 'Activer votre base de donnée privée' ?

Reply:
It's a DTrun, I won't change the translations, but I really want to 😢